packages/api-v2/src/student/student.controller.ts

```diff
@@ -45,6 +45,10 @@ export class StudentController {
       throw new BadRequestException();
     }
 
+    if (!isWithPlans) {
+      student.plans = [];
+    }
```

Comment:
makes it much easier to deal with an empty array than potentially undef on the frontend